lib/ok.ex

```diff
@@ -231,7 +251,21 @@ defmodule OK do
       end
     end
   end
+  defp nest([{:ok, _} = normal | []]) do
+    # last line of the with block is an ok literal
+    quote do
+      unquote(normal)
+    end
+  end
+  
+  defp nest([{{:., _, [{:__aliases__, _, [:OK]}, :success]}, _, _} = normal | []]) do
+    # last line of the with block is an OK.success macro line
+    quote do
+      unquote(normal)
+    end
+  end
   defp nest([normal | []]) do
+    # last line of the with block is a func
```

Comment:
> I think the most likely next case would be if someones last line is result where that variable includes the wrapping tuple.

Hmm, I'm not sure what you mean by this. Perhaps a gitter or slack chat could be profitable? I'm up early my time, 6 am CST (12 pm UTC) at the latest most days, and I only work on ibGib now, so I could make time at any point during the day to work with your schedule. If you need me up earlier, I could do 4 am CST (10 am UTC), but doing it late - say after 9 pm CST (3 am UTC) is where it gets tough for me.